packages/device_info/lib/device_info.dart

```diff
@@ -241,6 +244,39 @@ class IosDeviceInfo {
       localizedModel: json['localizedModel'],
       identifierForVendor: json['identifierForVendor'],
       isPhysicalDevice: json['isPhysicalDevice'] == 'true',
+      utsname: new IosUtsname._(
+        sysname: json['utsname.sysname'],
+        nodename: json['utsname.nodename'],
+        release: json['utsname.release'],
+        version: json['utsname.version'],
+        machine: json['utsname.machine'],
+      ),
     );
   }
 }
+
+/// Information derived from `utsname`.
+class IosUtsname {
+  IosUtsname._({
+    this.sysname,
+    this.nodename,
+    this.release,
+    this.version,
+    this.machine,
+  });
+
+  /// OS name.
+  final String sysname;
+
+  /// Network node name.
+  final String nodename;
+
+  /// Release level.
+  final String release;
+
+  /// Version level.
+  final String version;
+
+  /// Hardware type.
```

Comment:
Perhaps note that this value does not exactly correspond to public device model names and link to https://stackoverflow.com/a/11197770/1704813?